[GitHub issue 1095 + Experiments]: Customer confused about logging summary metrics (#1204)
wandb/docs: content/guides/models/track/log/log-summary.md

## Customize summary metrics

Custom summary metrics are useful for capturing model performance at the best step of training in your `wandb.summary`. For example, you might want to capture the maximum accuracy or the minimum loss value, instead of the final value.

> The default summary behavior uses the final value from history. You can customize summary metrics using the `summary` argument in `define_metric` which accepts the following values:

Reviewer: ```suggestion
By default, the summary uses the final value from history. To customize summary metrics, pass the `summary` argument in `define_metric`. It accepts the following values:
```

* `"min"`
* `"max"`
* `"mean"`
* `"best"`
* `"last"`
* `"none"`

> The `"best"` parameter can only be used in conjunction with the optional `objective` argument which accepts values `"minimize"` and `"maximize"`. 

Reviewer: ```suggestion
You can use `"best"` only when you also set the optional `objective` argument to `"minimize"` or `"maximize"`. 
```

> The following code snippet demonstrates how track the min and max values of loss and accuracy in the summary:

Reviewer: ```suggestion
This example adds the min and max values of loss and accuracy to the summary:
```

    wandb.log(log_dict)
```

## View summary metrics

> You can view summary values within a run's **Overview** page or within your project's **Run Table**.

Reviewer: ```suggestion
View summary values in a run's **Overview** page or or your project's **Run Table**.
```
Should "table" be in the bold?

{{< tabpane text=true >}}
{{% tab header="Run Overview" value="overview" %}}

> 1. Navigate to the W&P App.

Reviewer: ```suggestion
1. Navigate to the W&B App.
```

2. Select the **Workspace** tab.

> 3. From the list of runs, select the name of the run that logged the summary values.

Reviewer: ```suggestion
3. From the list of runs, click the name of a run that logged the summary values.
```

> 4. Click on the **Overview** tab.

Reviewer: ```suggestion
4. Select the **Overview** tab.
```

> 5. Scroll down to the **Summary** section to view the summary values.

Reviewer: ```suggestion
5. View the summary values in the **Summary** section.
```

> {{< img src="/images/track/customize_summary.png" alt="" >}}

Reviewer: Please add some alt text

{{% /tab %}}
{{% tab header="Run Table" value="run table" %}}

> 1. Navigate to the W&P App.

Reviewer: ```suggestion
1. Navigate to the W&B App.
```

2. Select the **Runs** tab.
3. Within the **Runs** table, you can view the summary values within the columns based on the name of the summary value.

{{% /tab %}}

{{% tab header="W&B Public API" value="api" %}}

You can use the W&B Public API to fetch the summary values of a run. 

> For example, the following code example demonstrates one way to retrieve the summary values logged to a specific run using the W&B Public API and Pandas:

Reviewer: ```suggestion
This code example demonstrates one way to retrieve the summary values logged to a specific run using the W&B Public API and pandas:
```
Nit: https://pandas.pydata.org/ is lowercase `pandas`

Author: Good catch!
